src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -174,7 +180,7 @@ public override SymbolKind Kind
         {
             get
             {
-                return SymbolKind.TupleType;
+                return _underlyingType.Kind;
```

Comment:
This should be SymbolKind.NamedType. Underlying could be an error type, but this symbol doesn't inherit from ErrorTypeSymbol. So, SymbolKind.NamedType is the only valid kind for it.